src/interactive-window/editor-integration/codewatcher.ts

```diff
@@ -1013,41 +1001,15 @@ export class CodeWatcher implements ICodeWatcher {
                 result = await interactiveWindow.addCode(code, file, line);
             }
         } catch (err) {
-            if (err instanceof InteractiveCellResultError) {
```

Comment:
We no longer display errors in cells from code watcher.
The IW will display the errors in the cells.